addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -2882,185 +2870,6 @@ QUnit.test('post a simple message', async function (assert) {
     );
 });
 
-QUnit.test('post message on non-mailing channel with "Enter" keyboard shortcut', async function (assert) {
-    assert.expect(2);
-
-    // channel expected to be found in the sidebar
-    // with a random unique id that will be referenced in the test
-    this.data['mail.channel'].records.push({ id: 20, mass_mailing: false });
-    await this.start({
-        discuss: {
-            params: {
-                default_active_id: 'mail.channel_20',
-            },
-        },
-    });
-    assert.containsNone(
-        document.body,
-        '.o_Message',
-        "should not have any message initially in channel"
-    );
-
-    // insert some HTML in editable
-    await afterNextRender(() => {
-        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
-        document.execCommand('insertText', false, "Test");
-    });
-    await afterNextRender(() => {
-        const kevt = new window.KeyboardEvent('keydown', { key: "Enter" });
-        document.querySelector('.o_ComposerTextInput_textarea').dispatchEvent(kevt);
-    });
-    assert.containsOnce(
-        document.body,
-        '.o_Message',
-        "should now have single message in channel after posting message from pressing 'Enter' in text input of composer"
-    );
-});
-
-QUnit.test('do not post message on non-mailing channel with "SHIFT-Enter" keyboard shortcut', async function (assert) {
```

Comment:
must stay as well

Reply:
Actually the title of this test is wrong, we write "mass_mailing: True" in the test (so it should be removed)

Reply:
I would prefer if you kept the test and made it work, because we do want a test to cover that shift+enter does not post, but adds a new line instead.